service-api/src/main/java/greencity/validator/EmailAddressValidator.java

```diff
@@ -0,0 +1,30 @@
+package greencity.validator;
+
+import greencity.constant.AppConstant;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.apache.commons.lang3.Validate;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * This class should validate email.
+ *
+ * @author Volodymyr Mladonov
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class EmailAddressValidator {
+    /**
+     * This method validate language.
+     *
+     * @param emailAddress to be validated
+     * @return true if everything is ok
+     */
+    public static boolean isValid(String emailAddress) {
+        Validate.notBlank(emailAddress, "Email address couldn't be blank");
+        Pattern regexPattern = Pattern.compile(AppConstant.VALIDATION_EMAIL);
+        Matcher regMatcher = regexPattern.matcher(emailAddress);
```

Comment:
emailAddress.matches()

Reply:
Sadly, but Roma disaproved this change: https://stackoverflow.com/questions/2469244/whats-the-difference-between-string-matches-and-matcher-matches

Reply:
This is the same implementation as string.matches(); You can move Pattern regexPattern = Pattern.compile(AppConstant.VALIDATION_EMAIL); to the static variable of the class, then the pattern will be precompiled, otherwise you compile this each time when methods are used;